ios/Helpers/RCCTitleViewHelper.m

```diff
@@ -194,10 +194,17 @@ -(UILabel*)setupTitle:(NSDictionary*)style
     
     UIFont *titleFont = [UIFont boldSystemFontOfSize:17.f];
     
-    id fontSize = style[@"navBarTitleFontSize"];
+    id fontSize = style[@"navBarTextFontSize"];
+    id fontBold = style[@"navBarTextFontBold"];
+    
     if (fontSize) {
         CGFloat fontSizeFloat = [RCTConvert CGFloat:fontSize];
-        titleFont = [UIFont boldSystemFontOfSize:fontSizeFloat];
+        
+        if (fontBold == 1) {
```

Comment:
I'm not sure I understand what this check is for. Do you mean `if (fontBold)` to indicate that the property is non-null?